File: 2390. Removing Stars From a String/Approach_2.java
class Solution {
    public String removeStars(String s) {

        int n = s.length();
        Stack<Character> st = new Stack<>();

        for (int i = 0; i < n; i++) {
            if (s.charAt(i) != '*') {
                st.push(s.charAt(i));
            } else {
                st.pop();
            }
        }

        String ans = "";

        while (!st.isEmpty()) {
            Character element = st.peek();
            st.pop();
            ans += element;
        }
        String reversed = new StringBuffer(ans).reverse().toString();
        return reversed;

    }
}

File: 4030. Check ASCII Palindromic/Approach_2.cpp
class Solution {
private:
    string getBinary(int val) {
        string ans = "";
        while (val >= 2) {
            ans = ans + to_string(val % 2);
            val = val / 2;
        }

        ans += to_string(val);

        if (ans.size() != 8) {
            int n = 8 - ans.size();
            while (n--) {
                ans += "0";
            }
        }

        reverse(ans.begin(), ans.end());

        return ans;
    }

public:
    bool isPalindromic(string s) {
        string ans = "";
        for (int i = 0; i < s.size(); i++) {
            int Asciival = s[i];
            string temp = getBinary(Asciival);
            ans += temp;
        }
        // Check for the pallindrome
        for (int i = 0; i < ans.size() / 2; i++) {
            if (ans[i] != ans[ans.size() - i - 1])
                return false;
        }
        return true;
    }
};